application/src/test/java/org/togetherjava/tjbot/jda/JdaTester.java

```diff
@@ -487,6 +495,110 @@ public JdaTester() {
         return ErrorResponseException.create(reason, new Response(null, -1, "", -1, Set.of()));
     }
 
+    /**
+     * Creates a MessageReceivedEvent
+     * <p>
+     * The method creates a MessageReceivedEvent from the input message and mock the message author
+     * from the input user.
+     *
+     * @param message the message to create an event for
+     * @param user the author from the message
+     * @return the created MessageReceivedEvent
+     */
+    public MessageReceivedEvent createMessageReceivedEvent(@NotNull Message message,
+            @NotNull User user) {
+        doReturn(user).when(message).getAuthor();
+        return new MessageReceivedEvent(jda, MESSAGE_RESPONSE_NUMBER, message);
+    }
+
+    /**
+     * Creates a message spy
+     *
+     * @return the created spy
+     */
+    public @NotNull Message createMessageSpy() {
+        return spy(Message.class);
+    }
+
+    /**
+     * Creates an attachment
+     *
+     * @param id id of the attachment
+     * @param url url of the attachment
+     * @param proxyUrl proxyUrl of the attachment
+     * @param fileName fileName of the attachment
+     * @param contentType contentType of the attachment
+     * @param description description of the attachment
+     * @param size size of the attachment
+     * @param height height of the attachment
+     * @param width width of the attachment
+     * @param ephemeral is the attachment ephemeral
+     * @return the created attachment
+     */
+    public Message.Attachment createAttachment(long id, String url, String proxyUrl,
+            String fileName, String contentType, String description, int size, int height,
+            int width, boolean ephemeral) {
+        return new Message.Attachment(id, url, proxyUrl, fileName, contentType, description, size,
+                height, width, ephemeral, jda);
+    }
```

Comment:
This is not a helper. This is a pure forward. It does not make attachment creation easier for the user. Also, it includes a lot of things that the user simply does not care about at all.

In majority of cases, all the user cares about is the url, fileName, contentType and description. You should hide the rest.

Either directly or by offering a builder for this which has well set defaults.